packages/browser/src/login/oidc/ClientRegistrar.spec.ts

```diff
@@ -276,12 +368,18 @@ describe("ClientRegistrar", () => {
     });
 
     it("saves dynamic registration information", async () => {
+      const currentTime = 1651281010000;
+      jest.useFakeTimers();
+      jest.setSystemTime(currentTime);
+
       const mockFetch = (jest.fn() as any).mockResolvedValueOnce(
         /* eslint-disable camelcase */
         new NodeResponse(
           JSON.stringify({
             client_id: "some id",
             client_secret: "some secret",
+            // Expire this client in 5 seconds:
+            client_secret_expires_at: currentTime / 1000 + 5,
```

Comment:
We may want a test for the behaviour when `client_secret_expires_at` is in the past, as the `registerClient` would fail in this case, which would likely blow up the code for ClientRegistrar in browser.